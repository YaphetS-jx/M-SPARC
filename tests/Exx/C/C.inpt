# Test: C #
CELL: 4.609 4.609 4.609 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
#EXCHANGE_CORRELATION: LDA_PW
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fd
#SMEARING: 0.001
SMEARING: 0.003675
#SMEARING: 0.3166810501
TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8
#NSTATES: 24
PRINT_FORCES: 1
PRINT_ATOMS: 1

EXX_FLAG: 1
