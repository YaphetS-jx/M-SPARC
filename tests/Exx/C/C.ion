#=========================
# format of ion file
#=========================
# ATOM_TYPE:   <atom type name> 
# PSEUDO_POT:  <path/to/pseudopotential/>
# N_TYPE_ATOM: <num of atoms of this type>
# ATOMIC_MASS: <mass of atom of this type> #(optional, for MD only)
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX: #(optional)
# <xrelax> <yrelax> <zrelax>
# ...

# this is a comment

ATOM_TYPE:   C                             # atom type 
N_TYPE_ATOM: 8                             # number of atoms of this type
PSEUDO_POT:  C.pot   # pseudopotential
COORD:                                     # coordinates follows
    0.135468338586080   4.527060551618890   4.541124967357384
    1.162309237740988   1.186082772370191   1.185719079997291
    0.147680330393084   2.263800063839690   2.228792345919109
    1.128666549615841   3.532376259713281   3.330845407906999
    2.194599151155803   0.071830873163142   2.151109562182273
    3.583694310794446   1.123618787573712   3.521542497859103
    2.364416364816039   2.196173530066426   0.075405353749758
    3.429894840924236   3.509419443227792   1.208311525635582