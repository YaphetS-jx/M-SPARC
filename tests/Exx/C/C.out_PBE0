***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 02-Nov-2020  Start time: 15:02:35                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 4.609000 4.609000 4.609000 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003675000
CHEB_DEGREE: 25
NSTATES: 24
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/C/C.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.307267 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Exx/C/C.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.07 7.07 7.07
Number of atoms of type 1          :  8
Estimated total memory usage       :  11.90 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.9371501303E+00        1.441E-01        0.517
2            -4.9572219739E+00        7.765E-02        0.136
3            -4.9660714328E+00        2.359E-02        0.142
4            -4.9669077002E+00        3.522E-03        0.139
5            -4.9669188386E+00        1.618E-03        0.138
6            -4.9669218788E+00        6.875E-04        0.136
Total number of SCF: 6     
1            -4.6177321065E+00        3.185E-03        19.349
2            -4.6178407392E+00        1.647E-03        6.536
3            -4.6179895317E+00        1.774E-04        6.628
4            -4.6179937434E+00        3.213E-05        6.629
5            -4.6179936547E+00        1.156E-05        6.801
6            -4.6179936053E+00        4.384E-06        6.735
7            -4.6179935922E+00        1.306E-06        6.907
8            -4.6179935743E+00        3.693E-07        6.827
Total number of SCF: 8     
1            -4.6186617588E+00        5.441E-05        22.083
2            -4.6186635486E+00        3.249E-05        7.405
3            -4.6186664970E+00        2.428E-06        7.437
4            -4.6186665459E+00        6.210E-07        7.536
Total number of SCF: 4     
1            -4.6186775848E+00        1.661E-06        23.162
2            -4.6186776297E+00        7.916E-07        7.196
3            -4.6186776965E+00        6.229E-08        7.208
Total number of SCF: 3     
1            -4.6186779451E+00        5.982E-08        23.242
2            -4.6186779519E+00        4.530E-08        7.379
3            -4.6186779430E+00        8.117E-08        7.384
Total number of SCF: 3     
1            -4.6186779485E+00        8.208E-08        24.088
2            -4.6186779440E+00        6.195E-08        7.895
3            -4.6186779464E+00        1.411E-07        7.880
Total number of SCF: 3     
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -4.6186779464E+00 (Ha/atom)
Total free energy                  : -3.6949423571E+01 (Ha)
Band structure energy              :  2.5323538107E+01 (Ha)
Exchange correlation energy        : -1.4839510031E+01 (Ha)
Self and correction energy         : -1.3384485533E+02 (Ha)
Entropy*kb*T                       : -3.8882884952E-07 (Ha)
Fermi level                        :  1.6189736419E+00 (Ha)
Average force                      :  6.6303421142E-01 (Ha/Bohr)
Maximum force                      :  9.3733835827E-01 (Ha/Bohr)
Time for force calculation         :  5.694 (sec)
