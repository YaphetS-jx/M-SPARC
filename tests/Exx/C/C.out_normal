***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 02-Nov-2020  Start time: 15:01:44                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 4.609000 4.609000 4.609000 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003675000
CHEB_DEGREE: 25
NSTATES: 24
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/C/C.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.307267 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Exx/C/C.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.07 7.07 7.07
Number of atoms of type 1          :  8
Estimated total memory usage       :  11.90 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.9371501303E+00        1.441E-01        0.719
2            -4.9572219739E+00        7.765E-02        0.163
3            -4.9660714328E+00        2.359E-02        0.183
4            -4.9669077002E+00        3.522E-03        0.146
5            -4.9669188386E+00        1.618E-03        0.159
6            -4.9669218788E+00        6.875E-04        0.161
7            -4.9669223563E+00        1.958E-04        0.141
8            -4.9669224280E+00        2.754E-05        0.136
9            -4.9669223746E+00        1.305E-05        0.135
10           -4.9669224010E+00        2.506E-06        0.135
11           -4.9669223865E+00        8.521E-07        0.135
Total number of SCF: 11    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -4.9669223865E+00 (Ha/atom)
Total free energy                  : -3.9735379092E+01 (Ha)
Band structure energy              :  2.6832491741E+01 (Ha)
Exchange correlation energy        : -1.9177042974E+01 (Ha)
Self and correction energy         : -1.3384485533E+02 (Ha)
Entropy*kb*T                       : -8.8604433354E-08 (Ha)
Fermi level                        :  1.6635938602E+00 (Ha)
Average force                      :  6.6704090539E-01 (Ha/Bohr)
Maximum force                      :  9.4665212566E-01 (Ha/Bohr)
Time for force calculation         :  1.220 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  5.057 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
