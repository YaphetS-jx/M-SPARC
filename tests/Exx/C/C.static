***************************************************************************
                            Atom positions                                 
***************************************************************************
Cartesian coordinates of atoms (Bohr):
      0.1354683386       4.5270605516       4.5411249674
      1.1623092377       1.1860827724       1.1857190800
      0.1476803304       2.2638000638       2.2287923459
      1.1286665496       3.5323762597       3.3308454079
      2.1945991512       0.0718308732       2.1511095622
      3.5836943108       1.1236187876       3.5215424979
      2.3644163648       2.1961735301       0.0754053537
      3.4298948409       3.5094194432       1.2083115256
