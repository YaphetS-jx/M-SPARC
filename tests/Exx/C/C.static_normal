***************************************************************************
                            Atom positions                                 
***************************************************************************
Cartesian coordinates of atoms (Bohr):
      0.1354683386       4.5270605516       4.5411249674
      1.1623092377       1.1860827724       1.1857190800
      0.1476803304       2.2638000638       2.2287923459
      1.1286665496       3.5323762597       3.3308454079
      2.1945991512       0.0718308732       2.1511095622
      3.5836943108       1.1236187876       3.5215424979
      2.3644163648       2.1961735301       0.0754053537
      3.4298948409       3.5094194432       1.2083115256
Atomic forces (Ha/Bohr):
     -0.2516432258      -0.1174861255       0.5325599585
     -0.3431971624      -0.0784672340      -0.8787541453
     -0.7073350550       0.2807585699       0.4074977297
      0.7252933773      -0.5317258955      -0.0255817945
      0.5253084901       0.2562162848       0.6431190171
     -0.1026997744      -0.1310364829      -0.3064003100
      0.0036235308       0.4843781396      -0.3200856122
      0.1506498192      -0.1626372564      -0.0523548433
