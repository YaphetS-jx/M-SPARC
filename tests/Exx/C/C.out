***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 03-Nov-2020  Start time: 17:07:40                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 4.609000 4.609000 4.609000 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.003675000
CHEB_DEGREE: 25
NSTATES: 24
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Exx/C/C.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.307267 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Exx/C/C.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.07 7.07 7.07
Number of atoms of type 1          :  8
Estimated total memory usage       :  11.90 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.8654145345E+00        1.535E-01        0.554
2            -4.8848855438E+00        8.258E-02        0.148
3            -4.8934421206E+00        2.521E-02        0.172
4            -4.8942599385E+00        3.824E-03        0.150
5            -4.8942712807E+00        1.746E-03        0.136
6            -4.8942742477E+00        7.546E-04        0.137
Total number of SCF: 6     
